CoreScriptsRoot/Modules/Stats/StatsAnnotatedGraph.lua

```diff
@@ -176,7 +176,11 @@ function StatsAnnotatedGraphClass:_recursiveGetOrderOfMagnitude(estimate, target
   return self:_recursiveGetOrderOfMagnitude(estimate*10.0, target)
 end
 
-function StatsAnnotatedGraphClass:_updateValue()
+function StatsAnnotatedGraphClass:UpdateValuesAndRender()
+  if not StatsUtils.PerformanceStatsShouldBeVisible() then 
```

Comment:
would it make sense to register/desregister listeners instead of doing if checks like this? Would be more performant to only make these calls if we need to